docs/manuals/deployment/docker/production.md

```diff
@@ -0,0 +1,79 @@
+## Prerequisites
+
+- Knowledge of Docker.
+- [Prepare your application for deployment](../prepare-taipy-for-deployment.md)
+- [Followed steps for development](development.md)
+
+
+## Production ready Dockerfile
+
+The following Dockerfile contains minimum configuration for put your application in production.
+
+This template assumes that you provide a `requirements.txt` file with all your dependencies.
+Please, update the Docker command `CMD` with corresponding file name and Flask app of your application
```

Comment:
"Please, update the Docker command `CMD` with corresponding file name and Flask app of your application"
->
"Make sure you  update the Docker command `CMD` with the appropriate file name and Flask settings for your application"